corehq/apps/reports/tasks.py

```diff
@@ -185,36 +185,25 @@ def build_form_multimedia_zip(
         export_id,
         zip_name,
         download_id,
-        export_is_legacy=False,  # always False
         user_types=None,
         group=None):
 
     form_ids = get_form_ids_having_multimedia(
-        domain,
-        app_id,
-        xmlns,
-        parse(startdate),
-        parse(enddate),
-        group=group,
-        user_types=user_types,
+        domain, app_id, xmlns, parse(startdate), parse(enddate),
+        group=group, user_types=user_types,
     )
-    properties = _get_export_properties(export_id)
+    forms_info = _get_form_attachment_info(domain, form_ids, export_id)
 
     if not app_id:
         zip_name = 'Unrelated Form'
-    forms_info = list()
-    for form in FormAccessors(domain).iter_forms(form_ids):
-        if not zip_name:
```

Comment:
This condition is only met on the first form, since `'unknown form'` is truthy.  This let me move it out of the loop.  Don't sweat it though, I dropped all this in a few commits anyways.